src/commands/add-formatting.js

```diff
@@ -0,0 +1,134 @@
+const lang = require('language-classifier');
+const hljs = require('highlight.js');
```

Comment:
So whats the differences between these two again and the benefit we get from using both rather then just 1 or the other?

Reply:
`hljs` is the one which supports many language and that is why it sometimes confuses simple JS/CSS/HTML codes for other languages that nobody ever heard of. (Overall `hljs` is good but it can sometimes suffer from a little accuracy problem) As for `language-classifier`, it only supports a handful of languages and quite accurate when it is inside it's comfort zone but panic when it finds languages it doesn't support.

Another thing to note here is, `hljs` gives two guesses and `language-classifier` gives only one guess. Normally, `hljs` is the one which most of the times detects correctly but it sometimes can't properly guess simple things like CSS and HTML, that's where `language-classifier` comes in.

Overall, I think a **lot** testing needs to be done.

Reply:
I looked a little into the `language-classifier` package, and it looks like it hasn't been maintained for **years**, nor can I find any source code, only the 0.0.1 version on npm (which interestingly, isn't mentioned in the package.json)

due to this, it might make sense to focus on using **just** the hljs lib (which also hasn't been added to the package.json) as its at least maintained, with a **ton** of community support in comparison.

Reply:
Yeah....I just checked it too. I think we should scrap `language-classifier` and just use `hljs`.

As for the packages not being linked in the `package.json`, I have fixed that.

Reply:
Removed `language-classifier` compleletely..